sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -76,12 +80,16 @@ class AzureCliTestCredential : public AzureCliCredential {
   {
   }
 
-  std::string GetOriginalAzCommand(std::string const& resource, std::string const& tenantId) const
+  std::string GetOriginalAzCommand(
+      std::string const& resource,
+      std::string const& tenantId,
+      std::string const& subscription) const
   {
-    return AzureCliCredential::GetAzCommand(resource, tenantId);
+    return AzureCliCredential::GetAzCommand(resource, tenantId, subscription);
   }
 
   decltype(m_tenantId) const& GetTenantId() const { return m_tenantId; }
+  decltype(m_subscription) const& GetSubscription() const { return m_subscription; }
```

Comment:
Would using `auto` as the return type here make sense instead? It would have the same effect but be much more straightforward.